### PR TITLE
update sln file to match what mentioned in orchestrator samples readme

### PR DESCRIPTION
**Pull request by tsuwandy**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

**Review by feich-ms — APPROVED** [review]

**Comment by tsuwandy** [comment]
@feich-ms, waiting for this PR to finish https://github.com/microsoft/botbuilder-dotnet/pull/4699/files, which should help resolve the errors from running orchestrator samples.

**Comment by tsuwandy** [comment]
@mrivera-ms , could you approve this PR?  Thank you!

**Comment by mrivera-ms** [comment]
@tsuwandy , you need approval by one of the code owners in the group bb-samples.

**Review by cleemullins — APPROVED** [review]

**Pull request merged**
